include/deal.II/base/polynomial_space.h

```diff
@@ -161,6 +163,16 @@ class PolynomialSpace
                               const Point<dim> &p) const;
 
   /**
+   * Computes the order @tparam order derivative of the <tt>i</tt>th
+   * polynomial at <tt>unit_point</tt>.
+   *
+   * Consider using compute() instead.
+   */
+  template <int order=1>
+  Tensor<order,dim> compute_derivative (const unsigned int i,
```

Comment:
default arguments for template arguments are not allowed in C++98. Can you remove the default value?